### PR TITLE
There is an error in the run_tf_ner.py script with the get_labels fun…

### PR DESCRIPTION
**Pull request by RodSernaPerez**
There is an error in the run_tf_ner.py scripts with the get_labels function (does not exist). It does not work with TPU. Both problems are fixed.

### CODE REVIEW

**Review by JetRunner — APPROVED** [review]
LGTM! The CI wants you to run `make style`

**Comment by jplu** [comment]
Thanks for the PR.

The issue with the `get_labels` has been fixed in a previous PR. Can you rebase on master and remove the `get_labels` part? We will keep the TPU fix :)

File: examples/token-classification/run_tf_ner.py [hunk 1]
```diff
@@ -277,5 +281,10 @@ def compute_metrics(p: EvalPrediction) -> Dict:
     return results
 
 
+def _mp_fn(index):
+    # For xla_spawn (TPUs)
+    main()
```
Comment (LysandreJik) [inline examples/token-classification/run_tf_ner.py:286]: `xla_spawn.py` is for pytorch/xla, it shouldn't work with a TF script.

Comment (jplu) [reply examples/token-classification/run_tf_ner.py:286]: Ok then can be claused.

**Comment by LysandreJik** [comment]
Closing as already implemented. Thanks a lot for your contribution @RodSernaPerez, looking forward to the next one!

**Pull request closed without merging**